### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by rickyhopkins** [framer/motion#612]
**1. Read the [FAQs](#faqs) 👇**

**2. Describe the bug**

When animating `borderRadius` with multiple shorthand values only the first value gets applied.
So this `borderRadius: "50% 0 50% 0"` becomes `borderRadius: "50%"` and `"0% 50% 0% 50%"` becomes `"0%"`

**3. IMPORTANT: Provide a CodeSandbox reproduction of the bug**

https://codesandbox.io/s/dazzling-wave-k973s?file=/src/App.js

**4. Steps to reproduce**

1. Go to codesandbox example
2. Click on the animatable example
3. Compare the shape to the expected shapes
4. They don't match.

**5. Expected behavior**

The animated element should animate between a border radius of `"50% 50% 0 0"` and `"0 0 50% 50%"`

**6. Video or screenshots**

N/A

**7. Environment details**

`"framer-motion": "^1.11.0",`

**Comment by sds-dubois** [comment]
FYI I tried with the last version (`3.3.0-beta.22`) and your example worked fine, I just had to provide an initial value in `baseStyle`.

However I found another bug with `borderRadius` animations when using `AnimateSharedLayout` (see this [example](https://codesandbox.io/s/festive-oskar-c4brs?file=/src/App.js:81-100)).
In one case, if I change `borderRadius` from `50px` to `10px`, the radius simply doesn't update. In another case, I switch from `"50% 50% 0 0"` to `"0 0 50% 50%"` and then it updates but without animation.

**Comment by mattgperry** [comment]
Apologises, `3.3.0-beta.22` was published by mistake, `3.3.0` should work as before (though I realise this also had a bug)

**Comment by sds-dubois** [comment]
Yep it still doesn't work with `3.3.0`. By the way, I had saved some changes by mistake in the code sandbox so it probably wasn't showcasing the right thing when you looked at it - but the example I mentioned is back now.
